ansible/roles/nginx/tasks/main.yml

```diff
@@ -1,18 +1,18 @@
 ---
 
-  - name: Download nginx release
+  - name: Download openresty/nginx bundle
```

Comment:
Yes , we need to remove any older existing nginx. Added steps/tasks to remove the old existing nginx ( /usr/local/nginx ).